jit-application/src/main/resources/application.yml

```diff
@@ -85,3 +85,12 @@ logging:
     org:
       hibernate:
         SQL: DEBUG
+
+camel:
+  max-redeliveries: 3
+  redelivery-delay: 1000
```

Comment:
that is only one second, perhaps 10000 would be a more appropiate retry delay

Reply:
Updated to 10000.